package.json

```diff
@@ -1,28 +1,39 @@
 {
   "name": "universal-search-content",
-  "version": "1.0.0",
-  "description": "the iframe that we put inside the autocomplete dropdown",
+  "version": "0.0.1",
+  "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "./node_modules/.bin/webpack --optimize-minimize --optimize-dedupe",
+    "start": "./node_modules/.bin/webpack-dev-server --content-base dist/",
```

Comment:
IIRC, npm adds ./node_modules/.bin/ to the $PATH, so you shouldn't need these prefixes in npm scripts.